releasenotes/notes/fix_dockerevent_nil_pointer-73e665aee0a222a1.yaml

```diff
@@ -0,0 +1,4 @@
+---
+fixes:
+  - |
+    Avoid processing of nil events.
```

Comment:
Changelog items should be undestandable without context, what about writing "Fix a nil-pointer segfault in docker event processing when an event is ignored"